lib/node_modules/@stdlib/stats/base/svariancewd/benchmark/benchmark.native.js

```diff
@@ -49,12 +51,8 @@ var opts = {
 */
 function createBenchmark( len ) {
 	var x;
-	var i;
 
-	x = new Float32Array( len );
-	for ( i = 0; i < x.length; i++ ) {
-		x[ i ] = ( randu()*20.0 ) - 10.0;
-	}
+	x = uniform( len, -10, 10, options);
```

Comment:
```suggestion
	var x = uniform( len, -10, 10, options);
```
we can simply declare and assign here, same comment for all other javascript benchmark files

Reply:
```suggestion
	var x = uniform( len, -10, 10, options );
```
spacing is also off here, please check for this in other JavaScript benchmarks too!